# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../netpbm/makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -urN netpbm-10.29.orig/converter/pbm/pbmtoppa/Makefile netpbm-10.29/converter/pbm/pbmtoppa/Makefile
--- netpbm-10.29.orig/converter/pbm/pbmtoppa/Makefile	2004-03-29 18:50:15.000000000 +0200
+++ netpbm-10.29/converter/pbm/pbmtoppa/Makefile	2005-09-05 13:15:08.714994500 +0200
@@ -22,6 +22,6 @@
 
 pbmtoppa: $(OBJECTS) $(NETPBMLIB) $(LIBOPT)
 	$(LD) $(LDFLAGS) -o pbmtoppa $(OBJECTS) \
-	  -lm `$(LIBOPT) $(NETPBMLIB)` $(LDLIBS) \
+	  `$(LIBOPT) $(NETPBMLIB)` $(LDLIBS) -lm \
 	  $(RPATH) $(LADD)
 
diff -urN netpbm-10.29.orig/Makefile.common netpbm-10.29/Makefile.common
--- netpbm-10.29.orig/Makefile.common	2005-08-15 00:03:22.000000000 +0200
+++ netpbm-10.29/Makefile.common	2005-09-05 13:14:46.473604500 +0200
@@ -264,7 +264,7 @@
 $(PORTBINARIES) $(MATHBINARIES): %: %.o $(NETPBMLIB) $(LIBOPT)
 # Note that LDFLAGS might contain -L options, so order is important.
 # LDFLAGS is commonly set as an environment variable.
-	$(LD) -o $@ $@.o $(MATHLIB) `$(LIBOPT) $(NETPBMLIB)` \
+	$(LD) -o $@ $@.o `$(LIBOPT) $(NETPBMLIB)` $(MATHLIB) \
 	  $(LDFLAGS) $(LDLIBS) $(RPATH) $(LADD)
 
 
